packages/atomic-react/rollup.config.mjs

```diff
@@ -1,31 +1,46 @@
-import typescript from '@rollup/plugin-typescript';
 import commonjs from '@rollup/plugin-commonjs';
 import {nodeResolve} from '@rollup/plugin-node-resolve';
 import replace from '@rollup/plugin-replace';
+import typescript from '@rollup/plugin-typescript';
+import {defineConfig} from 'rollup';
 import nodePolyfills from 'rollup-plugin-polyfill-node';
 import {terser} from 'rollup-plugin-terser';
 
+/** @type {import("rollup").GlobalsOption} */
+const globals = {
+  react: 'React',
+  'react-dom': 'ReactDOM',
+  'react-dom/client': 'ReactDOM',
+  'react-dom/server': 'ReactDOMServer',
```

Comment:
Rollup interprets `react-dom`, `react-dom/client` and `react-dom/server` as being completely different packages.